docs/SPEC.md

```diff
@@ -93,35 +90,20 @@ Examples:
 
 The following keywords are reserved and may not be used as identifiers:
 
-    and    import  not  return   option   test
-    empty  in      or   package  builtin
-
-[IMPL#256](https://github.com/influxdata/flux/issues/764) Add in and empty operator support
+    and import  option   if
+    or  package builtin  then
+    not return  testcase else exists
 
 #### Operators
 
 The following character sequences represent operators:
 
     +   ==   !=   (   )   =>
     -   <    !~   [   ]   ^
-    *   >    =~   {   }
-    /   <=   =    ,   :
+    *   >    =~   {   }   ?
+    /   <=   =    ,   :   "
```

Comment:
`?` and `"` are operators?

Reply:
The `?` shows up in type signatures for optional arguments. The `"` shows up for string literals. They are not operators in the sense of binary operators but they are still operators for the language. This list was taken from the scanner.rl implementation.